### PR TITLE
Setup tailwind with Parcel.js and NPM

### PR DESCRIPTION
**Pull request by hyeomans**

### CODE REVIEW

File: source/_posts/setup-tailwindcss_parcel_npm.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
+---
+title: Setup Tailwindcss with Parcel.js or npm.
+date: 2018-07-19
```
Comment (keogh) [inline source/_posts/setup-tailwindcss_parcel_npm.md]: Yesterda's date? I think it's better to edit the date to the merge date or the intended publishing date

File: source/_posts/setup-tailwindcss_parcel_npm.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+```
+
+Description by line:
+
+1. Create new directory, change `new_tailwind_project` to whaterver name you want.
+1. Create a `package.json` file
+1. Add required development dependencies with explicit version numbers, for example tailwind will be: `0.6.4`
+1. Create an empty postcss configuration file.
+1. Generate tailwind configuration file.
+1. Create `src` folder and the main CSS file where you can define your custom css.
```
Comment (keogh) [inline source/_posts/setup-tailwindcss_parcel_npm.md]: You have repeated chunks of text, here from line 106 to 124 is repeated previously from lines 63 to 80

File: source/_posts/setup-tailwindcss_parcel_npm.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+
+Hopefully now you can start customizing your live-reload or `dist` folder.
+
+### <a name="common-configuration"></a> Common configuration 
+
+** Postcss **
+
+Modify your `postcss.config.js` like this:
+
+```js
```
Comment (keogh) [inline source/_posts/setup-tailwindcss_parcel_npm.md:149]: little quesiton, using `js` works the same as `javascript` for highlighting code?

**Review by keogh — DISMISSED** [review]
Some comments, one of them we could say it's maybe a bug

File: source/_posts/setup-tailwindcss_parcel_npm.md (hunk 3 above)
Comment (hyeomans) [reply source/_posts/setup-tailwindcss_parcel_npm.md:149]: Yes

File: source/_posts/setup-tailwindcss_parcel_npm.md (hunk 2 above)
Comment (hyeomans) [reply source/_posts/setup-tailwindcss_parcel_npm.md]: Moved to common configuration to avoid repetition.

Comment (keogh) [reply source/_posts/setup-tailwindcss_parcel_npm.md]: ohhh, there was a reason why it was repeated

**Review by keogh — APPROVED** [review]

**Pull request merged**